tests/e2e_tests/pipelines/tf_hub/tf_hub_case_class.py

```diff
@@ -0,0 +1,27 @@
+from collections import OrderedDict
```

Comment:
please put a license:

```
# Copyright (C) 2018-2024 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

Reply:
we have a lot places with missed license.

Reply:
I put it in every .py file